src/point_one/fusion_engine/common/portability.h

```diff
@@ -11,28 +11,28 @@
 // - https://gcc.gnu.org/wiki/Visibility.
 // - https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/port_def.inc
 #if defined(_WIN32) || defined(_MSC_VER) || defined(__CYGWIN__)
-  #ifdef BUILDING_DLL
-    #ifdef __GNUC__
-      #define P1_EXPORT __attribute__ ((dllexport))
-    #else
-      #define P1_EXPORT __declspec(dllexport)
-    #endif
-  #else
-    #ifdef __GNUC__
-      #define P1_EXPORT __attribute__ ((dllimport))
-    #else
-      #define P1_EXPORT __declspec(dllimport)
-    #endif
-  #endif
-  #define P1_HIDDEN
+#  ifdef BUILDING_DLL
```

Comment:
does this really work?

Reply:
The spacing in the nested `ifdef`? Yep, you can put spaces before or after the `#`